packages/TypeDeclaration/tests/Rector/Property/PropertyTypeDeclarationRector/Fixture/doctrine_relation_imported.php.inc

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+use Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture\Product;
+use Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture\Car;
+use Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture\Entity as Entity;
+
+class DoctrineRelationImported
+{
+    /**
+     * @ORM\OneToMany(targetEntity="Product")
+     */
+    private $products;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="Car")
+     */
+    private $cars;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Entity\Company", inversedBy="userBuildings")
+     * @ORM\JoinColumn(name="company_id", referencedColumnName="id", nullable=true)
+     */
+    private $company;
+
+    /**
+     * @ORM\OneToOne(targetEntity="Entity\Company")
```

Comment:
Accoding to Doctrine docs this can't work, as it is in a different namespace.

See: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/annotations-reference.html#manytoone

![image](https://user-images.githubusercontent.com/924196/62427581-5ae96a00-b6f5-11e9-9d28-639d0f9fa597.png)